src/main/java/com/amuzil/omegasource/magus/skill/forms/Form.java

```diff
@@ -1,12 +1,17 @@
 package com.amuzil.omegasource.magus.skill.forms;
 
 import com.amuzil.omegasource.magus.registry.Registries;
+import com.amuzil.omegasource.magus.skill.conditionals.InputData;
+
+import java.util.LinkedList;
 
 public class Form {
     private String name;
+    private LinkedList<InputData> inputData;
```

Comment:
Why not leave the `InputData` mapping to the input module? This allows other devs to expand upon our forms and add new ways of activation, vs locking them in.